FunctionCalculator.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace CourseApp
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class FunctionCalculator
+    {
+        public double Calculate(double a, double b, double x)
+        {
+            double top = Math.Log(Math.Abs(Math.Pow(b, 2) - Math.Pow(x, 2)), a);
```

Comment:
Rename class into Human Readability

Reply:
we may leave this one for now, (`function` will be better name due to task definition - calculates result of the function)